test/passing/fun_decl.ml

```diff
@@ -12,47 +12,47 @@ class t =
   end
 
 let func
-    aaaaaaaaaaa
-    bbbbbbbbbbbbbb
-    ccccccccccccccccccc
-    ddddddddddddddddddddd
-    eeeeeeeeeeeeeee =
+   aaaaaaaaaaa
+   bbbbbbbbbbbbbb
+   ccccccccccccccccccc
+   ddddddddddddddddddddd
+   eeeeeeeeeeeeeee =
```

Comment:
I don't understand these changes, but am assuming that they are intended. Is there some discussion that motivates indenting the arguments an odd number of spaces, i.e. in between two natural "tab stops", and furthermore indents the arguments less than the function name?

These changes are nicely guarded, so I don't object to merging this, but just for the record I don't get it. :-)

Reply:
So @hhugo, approve this PR is it's really what you want.

Reply:
I don't get the indentation either but it comes directly from the JaneStreet fork so I assume they have a use for it. But maybe the indentation could be parameterized with another option like `indent-fun-args=N` ? (with N=3 for JaneStreet)